packages/webapp/components/layouts/ProfileLayout/v2.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { ReactElement, ReactNode } from 'react';
```

Comment:
With removal of index should we not just call this `index`?

Reply:
Good catch!